### PR TITLE
Fixed admin/user dashboard problem

### PR DESCRIPTION
**Pull request by alaanousir**

### CODE REVIEW

File: portal/app/account/views.py [hunk 1]
```diff
@@ -308,11 +308,11 @@ def unconfirmed():
     return render_template("account/unconfirmed.html")
 
 
-@account.route("/dashboard", methods=["GET", "POST"])
+@account.route("/", methods=["GET", "POST"])
 @login_required
-def dashboard():
+def index():
     """User dashboard page."""
-    return render_template("account/dashboard.html")
```
Comment (MohamedKasem99) [inline portal/app/account/views.py:333]:
```python
def dashboard():
    """User dashboard page."""
    if current_user.role.index == "admin":
        return render_template("account/admin-dashboard.html")
    return render_template("account/user-dashboard.html")
```

Comment (MohamedKasem99) [reply portal/app/account/views.py:333]: you can then simply add these 2 html files

File: portal/app/templates/account/demo.html [hunk 2]
```diff
@@ -0,0 +1,181 @@
+{% extends 'layouts/base.html' %}
```
Comment (MohamedKasem99) [inline portal/app/templates/account/demo.html:1]: why call this demo.html?

File: portal/app/account/views.py (hunk 1 above)
Comment (alaanousir) [reply portal/app/account/views.py:333]: There is already a function in admin/views.py to render the admin dashboard with. The views.py in that folder has all functions specific to the admin and has `@admin_required` . I believe that is the default, to have separate views.py for admin, account, and main folders.

**Pull request merged**